pages.nl/common/pio-remote.md

```diff
@@ -0,0 +1,33 @@
+# pio remote
+
+> Helper commando voor PlatformIO Remote Development.
+> `pio remote [command]` accepteert dezelfde argumenten als de lokale tegenhanger `pio [command]`.
+> Meer informatie: <https://docs.platformio.org/en/latest/core/userguide/remote/index.html>.
+
+- Toon alle actieve Remote Agents:
```

Comment:
Is there an appropriate translation for `remote agent`?

Reply:
I think the same as @sebastiaanspeck. The best idea is to keep `remote agent`. It is a pretty standard term in the IT in The Netherlands.